src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -220,6 +220,19 @@ export function transform(formConfig, form) {
 
   const filterRatedViewFields = formData => filterViewFields(formData);
 
+  const filterServicePeriods = formData => {
+    const { serviceInformation } = formData;
+    if (!serviceInformation || hasGuardOrReservePeriod(serviceInformation)) {
```

Comment:
Ooh, tricky. It _looks_ like [`hasGuardOrReservePeriod`](https://github.com/department-of-veterans-affairs/vets-website/blob/remove-webpack-from-metalsmith-cv/src/applications/disability-benefits/all-claims/utils.jsx#L84) takes the `formData`, not the `serviceInformation`, but it _lies_. Sneaky, sneaky.